### PR TITLE
Mime/Mute communion fix

### LINKED ISSUE #9861: Mime can Cult Commune without breaking Vow of Silence
**Problem Description**:
Mimes can can commune with the cult without breaking their vow of silence, making it a silent, safe way to talk with the cult.

**What did you expect to happen**: 
As a Mime, you're unable to talk or make most noise with your Vow of Silence intact. Thus, I expected to not be able to commune.

**What happened instead**:
I was able to commune without any whispering from my character, but the message still appeared in cult chat.

**Why is this bad/What are the consequences:**
While I'm not sure if it's important, I do believe that the cult whispers are one of the good ways to identify a cultist, and this could be a workaround. Additionally, this could work for mute characters, though I'm not sure.

**Steps to reproduce the problem**:
1. Be Mime Cultist
2. Commune
3. Watch your message appear silently!

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Nothing I can think of.

### PR DESCRIPTION
**Pull request by Shadow-Quill**
**What does this PR do:**

Makes it so the Mime (if vowed to be silent) and anyone who is mute cannot commune with the cult.
(How're you supposed to speak if you can't say abracadabra?)

This fixes #9861 

**Images of sprite/map changes (IF APPLICABLE):**
N/A

**Changelog:**
:cl:
fix: Mimes under the vow of silence/Mute people can no longer commune with the cult.
/:cl:

### CODE REVIEW

**Comment by Aurorablade** [comment]
There is no good reason to put a player at a disadvantage due to job choice. This is not a proper fix for the issue. For that matter, the 'issue' isn't really an 'issue' but a balance concern. A proper fix would be adding in some sort of tell for the ability use but not so much its a truly dead giveaway.

File: code/game/gamemodes/cult/cult_comms.dm [hunk 1]
```diff
@@ -17,9 +17,14 @@
 	cultist_commune(usr, input)
 	return
 
+
+
 /proc/cultist_commune(mob/living/user, message)
 	if(!message)
 		return
+	if((user.disabilities & MUTE) || user.mind.miming) //User can't be under a vow of silence or have the mute disability to commune.
+		to_chat(user, "You cannot speak!")
```
Comment (Birdtalon) [inline code/game/gamemodes/cult/cult_comms.dm]: This would look nicer with an appropriate span, consider `warning` maybe?

**Comment by variableundefined** [comment]
Is this really a good idea? Communion is kinda supernatural in the first place.

**Comment by Shadow-Quill** [comment]
@Aurorablade I'll think up a way of showing they're communing without stopping them from doing so (miming and mute people)

**Comment by Shadow-Quill** [comment]
Will make a new PR when I've figured a better solution.

**Pull request closed without merging**